presto-main/src/main/java/io/prestosql/sql/planner/OrderingScheme.java

```diff
@@ -50,12 +50,22 @@ public OrderingScheme(@JsonProperty("orderBy") List<Symbol> orderBy, @JsonProper
         this.orderings = ImmutableMap.copyOf(orderings);
     }
 
+    /**
+     * Gets the all ORDER BY clauses.
```

Comment:
```suggestion
     * Gets all the defined ORDER BY clauses.
```
Where are these clauses defined?

Reply:
Description added